app/src/main/java/com/example/recipeopedia/activities/RecipeListActivity.java

```diff
@@ -0,0 +1,83 @@
+package com.example.recipeopedia.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import com.codepath.asynchttpclient.AsyncHttpClient;
+import com.codepath.asynchttpclient.RequestHeaders;
+import com.codepath.asynchttpclient.RequestParams;
+import com.codepath.asynchttpclient.callback.JsonHttpResponseHandler;
+import com.example.recipeopedia.R;
+import com.example.recipeopedia.RecipeAdapter;
+import com.example.recipeopedia.models.Recipe;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Headers;
+
+public class RecipeListActivity extends AppCompatActivity {
+
+    public static final String TAG = "RecipeListActivity";
+    public static final String URL_PREFIX = "https://api.edamam.com/api/recipes/v2";
+    public static final String APP_ID = "ee1f2fe0";
+    public static final String APP_KEY = "ec8cceba866280e53c0c9f2300c17b1b";
```

Comment:
Something like App IDs and App Keys should ideally be put into a string resource file. Doing it this way also allows you to update these values in one place instead of X number of places